EcoSoapBank/Network Controller/GraphQLOperation.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  GraphQLOperation.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 8/31/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+/// Encapsulates the required info to perform a GraphQL operation.
+/// Handles generating the appropriate url request.
+enum GraphQLOperation {
```

Comment:
I think this solution definitely works. Alternatively, if you wanted the return type to be automatically/safely "filled in" for you, I think something like this might work: https://www.reddit.com/r/swift/comments/iggmon/a_different_return_type_for_different_enum_cases/